fedimint-testing/src/ln/mod.rs

```diff
@@ -55,9 +55,8 @@ pub trait LightningTest: ILnRpcClient {
             .unwrap())
     }
 
-    /// Returns the amount that the gateway LN node has sent
-    async fn amount_sent(&self) -> Amount;
-
```

Comment:
Don't we use this at all in tests? :open_mouth:

Reply:
I was surprised to learn that we don't use this at all. Perhaps leftover from a previous refactor